github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/config/GithubRestClient.java

```diff
@@ -0,0 +1,20 @@
+package be.xplore.githubmetrics.githubadapter.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestClient;
+
+@Configuration
+public class GithubRestClient {
```

Comment:
class schould maybe be called GithubRestClientConfiguration or Config?

Reply:
fair enough. all set